### PR TITLE
Notification group rename

### PR DESCRIPTION
**Pull request by hero101**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

**Review by techsmyth — DISMISSED** [review]
Makes sense from a code perspective.

File: src/migrations/1642159571969-pref-definition-groups-rename.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export class prefDefinitionGroupsRename1642159571969
+  implements MigrationInterface
+{
+  name = 'prefDefinitionGroupsRename1642159571969';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            update user_preference_definition set groupName = 'NotificationCommunityAdmin'
+            where type in ('NOTIFICATION_COMMUNICATION_DISCUSSION_CREATED_ADMIN', 'NOTIFICATION_COMMUNICATION_UPDATE_SENT_ADMIN', 'NOTIFICATION_APPLICATION_RECEIVED')
```
Comment (valentinyanakiev) [inline src/migrations/1642159571969-pref-definition-groups-rename.ts]:
```suggestion
            where type in ('NotificationCommunityDiscussionCreatedAdmin', 'NotificationCommunityUpdateSentAdmin', 'NotificationApplicationReceived')
```

File: src/migrations/1642159571969-pref-definition-groups-rename.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        `);
+    await queryRunner.query(`
+            update user_preference_definition set groupName = 'NotificationGlobalAdmin'
+            where type = 'NOTIFICATION_USER_SIGN_UP'
```
Comment (valentinyanakiev) [inline src/migrations/1642159571969-pref-definition-groups-rename.ts]:
```suggestion
            where type = 'NotificationUserSignUp'
```

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
Migration is not working, see code suggestions. You are searching by the enum key, and should be searching by the value.

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**